README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
```

Comment:
I don't think so - it's downloaded by `vagrant/deploy.sh` which is listed as a prerequisite and the environment variable is set by the preceding `./vagrant/environment.sh` command.